### PR TITLE
Use poetry

### PR DESCRIPTION
**Pull request by graham-dds**
https://www.pivotaltracker.com/n/projects/2160940

Since [Pipenv](https://github.com/pypa/pipenv/issues/4058) is no longer being maintained, this PR replaces it with [poetry](https://python-poetry.org/).

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -34,12 +33,14 @@ RUN apk update && \
         util-linux \
         yarn
 
+RUN python -m venv /.venv
```
Comment (dandds) [inline Dockerfile]: I don't think you actually need this addition. If you look at line 81, we're coping `/install/.venv/` to the location in the final build stage, which means that `poetry` went ahead and generated `/install/.venv` by itself. Try running the build again without this line.

File: script/include/run_setup [hunk 2]
```diff
@@ -35,9 +35,9 @@ fi
 rm -rf ./node_modules
 
 if [ "${CREATE_VENV}" = "true" ]; then
-  # Ensure pipenv is installed
-  if ! pipenv --version >/dev/null 2>&1 ; then
-    echo "ERROR: pipenv is malfunctioning or not present"
+  # Ensure poetry is installed
+  if ! poetry --version >/dev/null 2>&1 ; then
+    echo "ERROR: poetry is malfunctioning or not present"
```
Comment (dandds) [inline script/include/run_setup]: The script tries to check the Pipfile on line 44 below.

File: Dockerfile (hunk 1 above)
Comment (graham-dds) [reply Dockerfile]: I think you're right! Removed

File: script/include/run_setup (hunk 2 above)
Comment (graham-dds) [reply script/include/run_setup]: Good call --changed here: https://github.com/dod-ccpo/atst/pull/1467/files#diff-0092d782907521aa0f88bd9fc620d585R44

**Pull request merged**